t/basic.t

```diff
@@ -3,6 +3,7 @@ use warnings;
 use Test::More 0.96;
 use utf8;
 
+use lib 'lib';
```

Comment:
This shouldn't be here. The correct lib dir will be added by `make test` or `prove -l`.

Reply:
Removed.